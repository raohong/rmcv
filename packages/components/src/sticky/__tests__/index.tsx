import { render } from '@testing-library/react';
import { Sticky } from '..';

it('render correctly', () => {
  const tree = render(<Sticky />);

  expect(tree.asFragment()).toMatchSnapshot();
});
