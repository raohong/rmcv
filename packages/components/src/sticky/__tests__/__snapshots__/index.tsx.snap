// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<DocumentFragment>
  .emotion-0 {
  box-sizing: border-box;
}

<div
    class="RVuiSticky-root emotion-0"
  >
    <div
      class="RVuiSticky-content emotion-0"
    />
  </div>
</DocumentFragment>
`;
